### PR TITLE
INTGR-254

### PR DESCRIPTION
**Pull request by wajihqazi94**

### CODE REVIEW

**Comment by PCGeotab** [comment]
- All instances of `&nbsp;` should be replaced with JSX syntax, e.g. `{"}`
- Table styling should be different, you could utilize our existing table styling like so:
Screenshot 2023-12-14 154845
Screenshot 2023-12-14 154829
Screenshot 2023-12-14 154737

File: src/pages/hardware/developingAnIOX/MIMEProtocol.tsx [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+                        <td>MIME data</td>
+                        <td>x</td>
+                        <td>1</td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+        <p>[1] A sequence number of 0 is only used for the first packet. The sequence number increments by 1 for each subsequent packet. If the sequence number reaches 255 (0xFF) and more packets need to be sent, the sequence number must reset to a value of 1 and continue counting.</p>
+        <h3>MIME Data</h3>
+        <table>
```
Comment (jyalenkatian) [inline src/pages/hardware/developingAnIOX/MIMEProtocol.tsx:64]: This table needs to be in a `<div className="table-container">`

File: src/pages/hardware/developingAnIOX/MIMEProtocol.tsx [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+}, function(deviceError) {
+    console.error("Device search failed:", deviceError);
+});
+`}/>
+    </div>
+);
+
+const pageTitle: PageTitleProps = {
+    "title": "MIME Protocol",
+    "breadCrumbItems": ["Hardware", "Developing an IOX", "MIME Protocol"]
```
Comment (jyalenkatian) [inline src/pages/hardware/developingAnIOX/MIMEProtocol.tsx:272]:
```suggestion
    "breadCrumbItems": ["Hardware", "Guides", "MIME Protocol"]
```
There won't be a "Developing an IOX" section in the new site, so let's replace it with "Guides". Can you also update the folder name to 'guides'?

**Review by jyalenkatian — CHANGES_REQUESTED** [review]
@wajihqazi94 Two things left to fix.

**Pull request merged**